src/Phinx/Console/Command/Init.php

```diff
@@ -38,13 +38,14 @@
 class Init extends Command
 {
     const FILE_NAME = 'phinx';
-
+    private $name = 'init';
+    
     /**
      * {@inheritdoc}
      */
     protected function configure()
     {
-        $this->setName($this->getName() ?: 'init')
+        $this->setName($this->getName())
```

Comment:
why not just `$this->setName('init')`?
After all, this current getter setter thing is super weird.

Reply:
For compatibility

Reply:
What compatibility?